SPECS/mlnx-nfsrdma/mlnx-nfsrdma.spec

```diff
@@ -248,6 +248,9 @@ fi
 %endif
 
 %changelog
+* Wed Feb 19 2025 Chris Co <chrco@microsoft.com>
```

Comment:
Missing version & release. Need to find out why rpmdev-bumpspec didn't automatically add it...